setup.py

```diff
@@ -62,6 +62,7 @@ def read(*filenames, **kwargs):
         "tf-nightly-cpu",
         "tensorflow-gpu",
         "tensorflow-cpu",
+        "tensorflow-macos",
```

Comment:
I would either `git commit --amend --author=...` to give phzeller credit, or add a "Co-authored-by: " line to the commit message.

Reply:
Woops that's just an error in the rebasing

Reply:
Actually for me the author is already showing as `phzeller`, what is it saying on your end?

Reply:
Ah yeah, looks like you did it correctly, Github is just showing your icon instead of no icon, which is strange.